docs/Additional-Reading.md

```diff
@@ -44,7 +44,19 @@
 * [RxJS](https://xgrommx.github.io/rx-book/), an on-line book by @xgrommx
 * [Journey from procedural to reactive Javascript with stops](https://glebbahmutov.com/blog/journey-from-procedural-to-reactive-javascript-with-stops/) by Gleb Bahmutov
 
+# RxAndroid
+
+* [RxAndroid Github page](https://github.com/ReactiveX/RxAndroid)
+* [RxAndroid basics](https://medium.com/@kurtisnusbaum/rxandroid-basics-part-1-c0d5edcf6850)
```

Comment:
Could you please mark entries that contain only 1.x code with `**(1.x)**` and mixed ones with `**(1.x & 2.x)**`?

Reply:
Done